### ISSUE TITLE
AMCP - PLAY ON EMPTY LAYER

### ISSUE THREAD

**Issue by PeterAkakpo** [CasparCG/server#961]
### Expected Behaviour

anytime a **PLAY {channel}-{layer}**  command is sent to an empty layer, the response should be a **PLAY FAILED** instead of **OK**. the above command cannot be considered as being successfully executed command

### Current Behaviour

the system returns a status of **OK** (which shows the validity of the command), but which gives a wrong impression. when you issue **PLAY 1-0 filename** and filename does not exists, you get a different response (**PLAY FAILED**) and not **OK** even though the syntax of the command is valid.

### Steps to Reproduce 
execute PLAY 1-0 with layer 0 empty (no file loaded)
### Environment
* CasparCG Server version: 2.2.0.4
* Operating system: Windows 10

### Attachments

**Comment by jesperstarkar** [comment]
@ronag Had a comment about this earlier in another issue, which was closed. Robert claims that a `OK` response only acknowledges that the command is received and valid, not taken effect. This definition is a breaking change in the AMCP protocol, but that is the current implementation, if I understand 2.2 correct. Can you please add some light to this @ronag?

And @Julusian will your work on async handling of scheduled/batched commands help on the blocking nature of commands @ronag is working to resolve?

**Comment by ronag** [comment]
```PLAY 1-0``` is an invalid command and should not return OK.

**Comment by jesperstarkar** [comment]
No, PLAY is perfectly valid! If you have loaded or paused content on a layer, you can invoke playback with `PLAY 1-0`

**Comment by ronag** [comment]
@jesperstarkar: Yes, you are right. It's `LOAD 1-0` which is invalid.

**Comment by ronag** [comment]
I'm not sure what is "correct" for this issue. AMCP is a bit semantically broken as it is right now.

**Comment by jesperstarkar** [comment]
The original description of correct behaviour is desired. Everything I mentioned is unrelated, or partially related.
